mne/io/fieldtrip/tests/test_fieldtrip.py

```diff
@@ -0,0 +1,224 @@
+    raw_fiff_mne = get_raw_data(cur_system, drop_extra_chs=True)
+    if use_info:
+        info = get_raw_info(cur_system)
+        pytestwarning = {'expected_warning': None}
+    else:
+        info = None
+        pytestwarning = no_info_warning
+
+    cur_fname = os.path.join(test_data_folder_ft,
+                             'raw_%s.mat' % (version,))
+
+    if version == 'v73' and not _has_h5py():
+        with pytest.raises(ImportError):
+            mne.io.read_raw_fieldtrip(cur_fname, info)
+        return
+    with pytest.warns(**pytestwarning):
+        raw_fiff_ft = mne.io.read_raw_fieldtrip(cur_fname, info)
+
+    if cur_system == 'BTI' and not use_info:
+        raw_fiff_ft.drop_channels(['MzA', 'MxA', 'MyaA',
+                                   'MyA', 'MxaA', 'MzaA'])
+
+    if cur_system == 'eximia' and not use_info:
+        raw_fiff_ft.drop_channels(['TRIG2', 'TRIG1', 'GATE'])
+
+    # Check that the data was loaded correctly
+    check_data(raw_fiff_mne.get_data(),
+               raw_fiff_ft.get_data(),
+               cur_system)
+
+    # Check info field
+    check_info_fields(raw_fiff_mne, raw_fiff_ft, use_info)
+
+
+@testing.requires_testing_data
+def test_load_epoched_as_raw():
+    """Test whether exception is thrown when loading epochs as raw."""
+    test_data_folder_ft = get_data_paths('neuromag306')
+    info = get_raw_info('neuromag306')
+    cur_fname = os.path.join(test_data_folder_ft, 'epoched_v7.mat')
+
+    with pytest.raises(RuntimeError):
+        mne.io.read_raw_fieldtrip(cur_fname, info)
+
+
+@testing.requires_testing_data
+def test_invalid_trialinfocolumn():
+    """Test for exceptions when using wrong values for trialinfo parameter."""
+    test_data_folder_ft = get_data_paths('neuromag306')
+    info = get_raw_info('neuromag306')
+    cur_fname = os.path.join(test_data_folder_ft, 'epoched_v7.mat')
+
+    with pytest.raises(ValueError):
+        mne.io.read_epochs_fieldtrip(cur_fname, info, trialinfo_column=-1)
+
+    with pytest.raises(ValueError):
+        mne.io.read_epochs_fieldtrip(cur_fname, info, trialinfo_column=3)
+
+
+@testing.requires_testing_data
+def test_create_events():
+    """Test 2dim trialinfo fields."""
+    from mne.externals.pymatreader import read_mat
+
+    test_data_folder_ft = get_data_paths('neuromag306')
+    cur_fname = os.path.join(test_data_folder_ft, 'epoched_v7.mat')
+    original_data = read_mat(cur_fname, ['data', ])
+
+    new_data = copy.deepcopy(original_data)
+    new_data['trialinfo'] = np.array([[1, 2, 3, 4],
+                                      [1, 2, 3, 4],
+                                      [1, 2, 3, 4]])
+
+    with pytest.raises(ValueError):
+        _create_events(new_data, -1)
+
+    for cur_col in np.arange(4):
+        evts = _create_events(new_data, cur_col)
+        assert np.all(evts[:, 2] == cur_col + 1)
+
+    with pytest.raises(ValueError):
+        _create_events(new_data, 4)
+
+
+@testing.requires_testing_data
```

Comment:
Travis suggests that this needs requires_h5py decorator

Reply:
yes, he does insist... ;-)